release_tools/semverup.py

```diff
@@ -202,12 +202,17 @@ def determine_new_version_number(project, current_version, pre_release):
 
     bump_patch = False
     bump_minor = False
+    bump_major = False
 
     for entry in entries.values():
-        if entry.category != CategoryChange.FIXED:
-            bump_minor = True
+        if entry.category in (CategoryChange.CHANGED, CategoryChange.REMOVED):
```

Comment:
It should only increase the major version when the current version is >= 1.x (not on 0.x versions).